### PR TITLE
Fixed some of the sentence structure

### PR DESCRIPTION
**Pull request by rv-bsnyder**
Fixed wrong preposition in the description for `base` and moved it to the correct place in the sentence. Made a few other minor edits to make this section a little stronger.

**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ sidebar: auto
 - Type: `string`
 - Default: `/`
 
-The base URL the site will be deployed at. You will need to set this if you plan to deploy your site under a sub path, for example GitHub pages. If you plan to deploy your site to `https://foo.github.io/bar/`, then `base` should be set to `"/bar/"`. It should always start and end with a slash.
+The base URL to which the site will be deployed. You will need to set this if you plan to deploy your site under a sub path, for example GitHub pages. If you plan to deploy your site to `https://foo.github.io/bar/`, then `base` should be set to `"/bar/"`. It should always start and end with a forward slash (/).
 
 The `base` is automatically prepended to all the URLs that start with `/` in other options, so you only need to specify it once.
 
-**Also see:**
+**See also:**
```
Comment (ulivz) [inline docs/config/README.md]: Both are acceptable, but since `See also` is used throughout the document, so no change is recommended here.

Comment (hzgotb) [reply docs/config/README.md]: 如果英文给不懂中文的外国人看的话，显然See also更符合英语的阅读习惯。而不是使用中国式英语Also see

Comment (ulivz) [reply docs/config/README.md]: 所以为什么只改这一处？为何不全部一起改了？

Comment (hzgotb) [reply docs/config/README.md]: 可能他只看到这一处？那就让他全部改呗。我只是提个建议而已。

**Pull request merged**